### ISSUE TITLE
Menu and side panel cover the screen on mobile

### ISSUE THREAD

**Issue by VirginiaBalseiro** [dokieli/dokieli#510]
When displaying the webpage at 320 pixels wide, the pop-ups overlap and cover the entire screen. As a result, the content of both the pop-ups and the page itself is no longer visible, and users cannot interact with other elements.

<img width="408" height="855" alt="Image" src="https://github.com/user-attachments/assets/5255726d-f24a-455f-a90c-053df39cf158" />

See: https://www.w3.org/WAI/WCAG22/Understanding/reflow
